db/db_universal_compaction_test.cc

```diff
@@ -1044,7 +1046,7 @@ TEST_P(DBTestUniversalCompaction, UniversalCompactionTrivialMoveTest2) {
   options.num_levels = 15;
   options.write_buffer_size = 100 << 10;  // 100KB
   options.level0_file_num_compaction_trigger = 8;
-  options.max_background_compactions = 1;
+  options.max_background_compactions = 2;
```

Comment:
Because I previously tried to restrict it to 1 to fix the test but it doesn't seem to be the cause of the problem: https://github.com/facebook/rocksdb/commit/1d9dbef64e6101e995035cfcf765255898a6bac5

I'm reverting it to get back the coverage.